server/src/http/routes/appointmentRequest.controller.ts

```diff
@@ -173,6 +179,35 @@ export default ({ users, appointments, mailer, widgetParameters, etablissements
     })
   )
 
+  router.post(
+    "/reply",
+    tryCatch(async (req, res) => {
+      await appointmentReplySchema.validateAsync(req.body, { abortEarly: false })
+      const paramsAppointementItem = req.body
+
+      const appointment = await appointments.getAppointmentById(paramsAppointementItem.appointmentId)
+      const user = await users.getUserById(appointment.candidat_id)
```

Comment:
```suggestion
      const appointment = await appointments.getAppointmentById(paramsAppointementItem.appointmentId)
      const user = await users.getUserById(appointment.candidat_id)
```

You can use a await Promise.all() here

Reply:
le pormise.all peut être utilisé sur les deux appels dépendant du resultat de `appointment`